src/nvim/menu.c

```diff
@@ -178,56 +178,56 @@ void ex_menu(exarg_T *eap)
    * If there is only a menu name, display menus with that name.
    */
   if (*map_to == NUL && !unmenu && enable == kNone) {
-    show_menus(menu_path, modes);
+    show_menus((char_u *)menu_path, modes);
```

Comment:
would it make sense to change the signature/internals etc of `show_menu`, `add_menu_path` specifically for `menu_path` like vars to reduce the amount of newly introduced casts?

Reply:
I tried to minimize the number of casts as much as possible. Changing the signature of show_menu make it require more casts :(.